Demo/Demo/SwiftUIComponents/CardPaymentViews/OrderCreateCardResultView.swift

```diff
@@ -1,6 +1,6 @@
 import SwiftUI
 
-struct OrderCreateCardPaymentResultView: View {
+struct OrderCreateCardResultView: View {
```

Comment:
Basically we have result views for each Orders API calls to check if they are working as expected.
so there are OrderCreateCardResultView, OrderCreatePayPalWebResultView, etc.
And there are also container views since I didn't want them to be all on one scrollable page.

Reply:
It sounds like this will be a great thing to refactor or can be shared in some way across payment methods since most of the logic is the same. I think that will help with naming as then we can have a singular `OrderResultView`.

Reply:
That is what I want to investigate in the refactor ticket. How to organize the different states to minimize views where it makes sense.